### ISSUE TITLE
AABB.combine() throws TypeError 

### ISSUE THREAD

**Issue by AlexGustafsson** [shakiba/planck.js#76]
## TLDR;

I'm trying to combine some AABBs to calculate the bounding box of a body's fixtures. However, I get `TypeError: undefined is not an object (evaluating 'b.lowerBound')`.

## Issue
https://github.com/shakiba/planck.js/blob/44aee932e5f3d972da6d93cbf549d3561ed4d859/lib/collision/AABB.js#L87-L103

The source states that one can combine one or two AABBs, but since no consideration is ever taken whether or not `b` is ever set - one will get `TypeError: undefined is not an object (evaluating 'b.lowerBound')` when trying to use only `a`. Box2d offers an option to only combine two AABBs, hence my guess that this is a bug.

A workaround is to call `a.combine(a, b)` which will work as expected. This could potentially just be a misinterpretation of the docs, but I thought it was worth to mention since it seems to differ from the Box2d implementation.

**Comment by zOadT** [comment]
You are right, an older version of this method starts with `b = b || this;`.

@shakiba is this an intended change or a bug? (If it is intended we have to update the definiton files)

**Comment by shakiba** [comment]
I don’t remember why I removed it, probably regression bug.
